cypress/e2e/additems.spec.cy.js

```diff
@@ -0,0 +1,49 @@
+const { baseUrl } = require('../variables/makeup_vars.js');
+
+describe('Добавление товаров в корзину', () => {
+  beforeEach(() => {
+    // Открываем URL маркетплейса
```

Comment:
Bad practice leave comments. Only when smth goes wrong. And better to use comment syntax(XXX, TODO)

Reply:
Fixed